ideascube/serveradmin/views.py

```diff
@@ -167,3 +170,41 @@ def wifi_history(request):
     return render(
         request, 'serveradmin/wifi_history.html',
         {'wifi_list': wifi_list.values()})
+
+
+@staff_member_required
+def packages(request):
+    # XXX: Should have a local Kinto cache.
+    client = Client(server_url='http://kinto.bsf-intranet.org/v1',
+                    bucket='ideascube', collection='packages')
+    packages = client.get_records()
+
+    return render(
+        request, 'serveradmin/packages.html',
+        {'packages': packages})
+
+
+@staff_member_required
+def package_detail(request, package_id):
+    client = Client(server_url='http://kinto.bsf-intranet.org/v1',
```

Comment:
The server might not have access to an external repository here. Thhis brings a use case of an offline-first client for Kinto.py (which is good) but also means that this PoC doesn't handle the offline case just yet.